modules/server/src/network/aggregations/AggregationAccumulator.ts

```diff
@@ -1,41 +1,33 @@
 import { SUPPORTED_AGGREGATIONS } from '../common';
 import { Aggregations, Bucket, NumericAggregations } from '../types/aggregations';
 import { AllAggregations } from '../types/types';
-import { RequestedFieldsMap } from '../util';
 
 type ResolveAggregationInput = {
 	aggregationsMap: AllAggregations;
-	requestedAggregationFields: string[];
 	accumulator: AllAggregations;
 };
 
+type AggregationsTuple = [Aggregations, Aggregations];
+type NumericAggregationsTuple = [NumericAggregations, NumericAggregations];
+
 /**
  * Resolves returned aggregations from network queries into single accumulated aggregation
  *
  * @param
  */
-const resolveAggregations = ({
-	aggregationsMap,
-	requestedAggregationFields,
-	accumulator,
-}: ResolveAggregationInput) => {
-	requestedAggregationFields.forEach((fieldName) => {
+const resolveAggregations = ({ aggregationsMap, accumulator }: ResolveAggregationInput) => {
+	Object.keys(aggregationsMap).forEach((fieldName) => {
 		const aggregation = aggregationsMap[fieldName];
 		const aggregationType = aggregation?.__typename || '';
-		const accumulatedFieldAggregation = accumulator[fieldName];
 
-		if (aggregation && accumulatedFieldAggregation) {
-			const resolvedAggregation = resolveToNetworkAggregation(aggregationType, [
-				aggregation,
-				accumulatedFieldAggregation,
-			]);
+		const accumulatedFieldAggregation = accumulator[fieldName];
 
-			// mutation - update a single aggregations field in the accumulator
-			accumulator[fieldName] = resolvedAggregation;
-		}
+		// mutation - update a single aggregations field in the accumulator
+		// if first aggregation, nothing to resolve yet
+		accumulator[fieldName] = !accumulatedFieldAggregation
+			? aggregation
+			: resolveToNetworkAggregation(aggregationType, [aggregation, accumulatedFieldAggregation]);
 	});
```

Comment:
clean up resolution calling. if there's no existing aggregation, nothing to resolve with